ui/webui/resources/br_elements/br_toolbar/br_toolbar_search_field.html

```diff
@@ -21,6 +21,10 @@
       :host:not([hidden]) {
         display: flex;
       }
+      
```

Comment:
nitpick: can you remove the spaces here? 😄

Reply:
fixed, thanks :)